### DISCUSSION TITLE
Non docker install documentation ?

### DISCUSSION THREAD

**Discussion by ithakaa** [hackershare/hackershare#124] [Ideas]
Hi

Thanks so much for all your efforts on hackershare, I'm wondering if you can update the installation document with instructions on how to install on Ubuntu without the need for docker.

It would be greatly appreciated.

**Comment by ithakaa** [comment]
does anyone have any documentation to install this without docker ?

**Reply by hooopo** [reply]
why not docker? what os do you use?

**Reply by ithakaa** [reply]
Really looking forward to installing it on Ubuntu

**Reply by hooopo** [reply]
OK

**Reply by ithakaa** [reply]
So no docs.
Ok

**Reply by ithakaa** [reply]
Anyone have any suggestions on installing this without docker?

A simple how-to would be great

**Reply by ithakaa** [reply]
So I guess that's a no for installation notes to install it without docker?

**Reply by songhuangcn** [reply]
> So I guess that's a no for installation notes to install it without docker?

Can this help you?

https://github.com/hackershare/hackershare/blob/main/bin/deploy-setup-ubuntu

This is recorded by me configuring the production environment.
